presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedTestBase.java

```diff
@@ -304,4 +354,212 @@ private void assertExplainAnalyze(@Language("SQL") String query)
 
         assertTrue(value.matches("(?s:.*)CPU:.*, Input:.*, Output(?s:.*)"), format("Expected output to contain \"CPU:.*, Input:.*, Output\", but it is %s", value));
     }
+
+    @Test
+    public void testTableWithDeleteFile()
+            throws Exception
+    {
+        List<String> fileFormats = ImmutableList.of("PARQUET", "ORC");
+        for (String fileFormat : fileFormats) {
+            testTableWithPositionDelete(fileFormat);
+            testTableWithEqualityDelete(fileFormat);
+            testTableWithPositionDeleteAndEqualityDelete(fileFormat);
+            testTableWithPositionDeletesAndEqualityDeletes(fileFormat);
+        }
+    }
+
+    private void testTableWithPositionDelete(String fileFormat)
```

Comment:
Please make these tests public and add `@Test` annotation so they get picked up

Reply:
@tdcmeehan Fixed.